README.md

```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
 
 ## Building the Code
 
-1. Clone the repository using HTTP: git clone https://github.com/openMF/mobile-wallet.git
-
-2. Open Android Studio.
-
+1. Fork the repository.
+2. Go to your fork and clone only the dev branch using `git clone git clone -b <branch> <remote_repo>`
```

Comment:
Why `git clone` is appearing twice? Also, if you want the developer to clone only dev branch then you should replace `<branch>` with `dev`.

Reply:
I didn't notice the git clone coming twice. Changes done